webapp/src/cli.rs

```diff
@@ -74,6 +78,15 @@ impl Opts {
         Ok(data_dir)
     }
 
+    pub fn cert_dir(&self) -> Result<PathBuf> {
+        let cert_dir = match self.cert_dir.clone() {
+            None => current_dir()?.join("webapp/certs"),
```

Comment:
So if we don't specify and this fallback directory doesn't exist, we would just fail, right?

I guess this fallback is convenient if we run the `webapp` from the root of this repo.

Reply:
Exactly! 👍